flow2ml/Flow.py

```diff
@@ -93,6 +94,32 @@ def applyFilters(self,filters):
       progress_i = 0
     print()
 
+  def applyAugmentation(self,operations):
+    ''' 
+      Applies given data augmentation operations 
+      Args : 
+        operations : (dictonary) python dictionary containing key value pairs of operations and values to be applied to the image data.
```

Comment:
Please specify type of operations with respect to values

Reply:
> @yvkrishna, could you please review?

I think It would be better if it is in this way

***Horizontal Flip***
`augmentation_techniques = { flipping:'horizontal' }`

***Vertical Flip***
`augmentation_techniques = { flipping:'vertical' }`

***Cross Flip***
`augmentation_techniques = { flipping:'cross' }`